polyfills/fm/client/fmradio.js

```diff
@@ -0,0 +1,553 @@
+        if (answer.error) {
+          self._fireError(answer.error);
+        } else {
+          self._fireSuccess(answer.result);
+        }
+      }
+    };
+  };
+
+  var serializeRequest = function(data) {
+    return {
+      id: ++_currentRequestId,
+      data: data,
+      processAnswer: function(answer) {
+        if (answer.error) {
+          // handle error properly
+        } else {
+          window.navigator.mozFMRadio[answer.result.name] = answer.result.value;
+        }
+      }
+    };
+  };
+
+  var enabled, antennaAvailable, frequencyLowerBound, frequencyUpperBound,
+    channelWidth, frequency;
+
+  window.navigator.mozFMRadio = {
+    fakeEnabled: false,
+
+    fakeAntennaAvailable: false,
+
+    fakeFrequencyLowerBound: 87.5,
+
+    fakeFrequencyUpperBound: 108,
+
+    fakeChannelWidth: 0.1,
+
+    disable: function fm_disable() {
+      var request = new FakeDOMRequest();
+      var data = {
+        operation: 'invoke',
+        method: 'disable'
+      };
+
+      request.serialize = serializeDOMRequest.bind(request, data);
+      navConnPromise.then(conn => conn.sendObject(request));
+      return request;
+    },
+
+    enable: function fm_enable(frequency) {
+      var request = new FakeDOMRequest();
+      var data = {
+        operation: 'invoke',
+        method: 'enable',
+        params: frequency
+      };
+
+      request.serialize = serializeDOMRequest.bind(request, data);
+      navConnPromise.then(conn => conn.sendObject(request));
+      return request;
+    },
+
+    seekUp: function fm_seekUp() {
+      var request = new FakeDOMRequest();
+      var data = {
+        operation: 'invoke',
+        method: 'seekUp'
+      };
+
+      request.serialize = serializeDOMRequest.bind(request, data);
+      navConnPromise.then(conn => conn.sendObject(request));
+      return request;
+    },
+
+    seekDown: function fm_seekDown() {
+      var request = new FakeDOMRequest();
+      var data = {
+        operation: 'invoke',
+        method: 'seekDown'
+      };
+
+      request.serialize = serializeDOMRequest.bind(request, data);
+      navConnPromise.then(conn => conn.sendObject(request));
+      return request;
+    },
+
+    cancelSeek: function fm_cancelSeek() {
+      var request = new FakeDOMRequest();
+      var data = {
+        operation: 'invoke',
+        method: 'cancelSeek'
+      };
+
+      request.serialize = serializeDOMRequest.bind(request, data);
+      navConnPromise.then(conn => conn.sendObject(request));
+      return request;
+    },
+
+    setFrequency: function fm_setFrequency(freq) {
+      var request = new FakeDOMRequest();
+      var data = {
+        operation: 'invoke',
+        method: 'setFrequency',
+        params: freq
+      };
+
+      request.serialize = serializeDOMRequest.bind(request, data);
+      navConnPromise.then(conn => conn.sendObject(request));
+      return request;
+    }
+  };
+
+  Object.defineProperty(window.navigator.mozFMRadio, 'enabled', {
```

Comment:
Since you're defining mozFMRadio as a object directly, you can just add those properties to previous definition, without the Object.define,....

...
get enabled() {
...
},